python/sparkdl/transformers/tf_tensor.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import, division, print_function
+
+import logging
+import numpy as np
+import tensorflow as tf
+import tensorframes as tfs
+
+from pyspark.ml import Transformer
+from pyspark.ml.param import Param, Params
+from pyspark.sql.functions import udf
+
+from sparkdl.graph.builder import IsolatedSession
+import sparkdl.graph.utils as tfx
+from sparkdl.transformers.param import (
+    keyword_only, HasInputMapping, HasOutputMapping, SparkDLTypeConverters,
+    HasTFGraph, HasTFHParams)
+
+__all__ = ['TFTensorTransformer']
+
+logger = logging.getLogger('sparkdl')
+
+class TFTensorTransformer(Transformer, HasTFGraph, HasTFHParams, HasInputMapping, HasOutputMapping):
```

Comment:
TFTensorTransformer is redundant since it spells out to be TensorFlowTensorTransformer. Why not just TFTransformer or TFModel?